From 97bc0e11d923f14f6cb36ad6af1ef77438dd5133 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Sat, 10 Nov 2018 19:36:37 -0700
Subject: [PATCH 2/2] dgit(1): drop warning about SOURCE_ONLY_CHANGES=yes in
 pbuilderrc

dgit now overrides this configuration option if the user happens to
have set it.

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit.1 | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/dgit.1 b/dgit.1
index c3dd420..47b436b 100644
--- a/dgit.1
+++ b/dgit.1
@@ -215,11 +215,6 @@ option.  If you want to pass other options to pbuilder, use the
 (remember that dgit options should appear between \fBdgit\fR and
 \fBpbuilder\fR).
 
-You should ensure that in your pbuilderrc you do
-.B not
-have the setting
-.B SOURCE_ONLY_CHANGES=yes
-as this may cause trouble.
 .TP
 \fBdgit cowbuilder\fR [\fIdebbuildopts\fP]
 Like \fBdgit pbuilder\fR, but uses
-- 
2.11.0

